app/[localeCode]/error.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable id-blacklist */
+
+'use client';
+
+import Illustration from '@/public/images/error.svg';
+
+import 'modern-normalize/modern-normalize.css';
+import '@/styles/styles.globals.scss';
+
+import styles from './error.module.scss';
+
+export default function Error() {
```

Comment:
Hmm.. I wonder if it's possible to only display the error page and not the rest like the header etc.? 🤔

Reply:
It uses the folder layout component, not sure of a way to bypass that yet

Reply:
What about this? https://nextjs.org/docs/pages/building-your-application/routing/custom-error#customizing-the-500-page

Reply:
What if you move the `error.tsx` file directly into `app` folder? Does that work or not?

Reply:
Tried but that also does not work

Reply:
Hmm, okay. How did you simulate the error for the screenshot? I'm wondering how it's going to look if all of the API request return a 500 for example.

Reply:
I did this:

<img width="742" alt="image" src="https://github.com/user-attachments/assets/a6b8e1f6-b5d1-49bf-9b34-f84741287460" />

Reply:
Right, but that is an error on a specific page. Can you throw the error somewhere else, maybe in the main layout component?

I just want to see how it works if the error happens outside of the actual page (before the app can be even rendered).

Reply:
It doesn't work, fixing with this: https://nextjs.org/docs/app/api-reference/file-conventions/error#global-error